sectionsUtilityTool/src/main/webapp/error.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Error page for Sections Manager</title>
+  </head>
+  <body class="container-fluid" style="padding-top:20px;">    
+  	<h1>Acccess denied!</h1>
```

Comment:
Dave, this is a clear case of 403 since he is authenticated but not in the access list of our tool. so he is not authorized in that perspective he is denied access.